syntax = "proto2";

message Robots_Status{
  repeated Robot_Status robots_status = 1;
}

message Robot_Status {
  required int32 robot_id  = 1;
  required bool infrared   = 2;
  required bool flat_kick  = 3;
  required bool chip_kick  = 4;
  optional string login_name = 5;
  optional float wheel1 = 6;
  optional float wheel2 = 7;
  optional float wheel3 = 8;
  optional float wheel4 = 9;
}
